src/data/finset/pointwise.lean

```diff
@@ -1070,3 +1096,74 @@ by { simp_rw ←image_neg, exact image₂_image_left_comm neg_smul }
 
 end ring
 end finset
+
+open_locale pointwise
+
+namespace set
+section has_one
+variables [has_one α]
+
+@[simp, to_additive] lemma to_finset_one : (1 : set α).to_finset = 1 := rfl
+
+@[simp, to_additive]
+lemma finite.to_finset_one (h : (1 : set α).finite := finite_one) : h.to_finset = 1 :=
+finite.to_finset_singleton _
+
+end has_one
+
+section has_mul
+variables [decidable_eq α] [has_mul α] {s t : set α}
+
+@[simp, to_additive] lemma to_finset_mul (s t : set α) [fintype s] [fintype t] [fintype ↥(s * t)] :
```

Comment:
Why not `mul_to_finset`? Similarly elsewhere.

Reply:
`to_finset` is the outer function.